src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+body {
+  margin: 0;
+  --linkColor: #00acdc;
```

Comment:
It's better to use kebab-case for all CSS: `--link-color`. Or better `--color-link`.